### PR TITLE
Article: "Many to Many Relationship with ABP and EF Core"

### LINKED ISSUE #9975: Create a many to many relationship example with ABP and EF Core
Users are having issue when trying to create a basic many-to-many EF Core relationship. 
Make a simple sample with ABP (open-source) and write a step by step document.
reported at https://support.abp.io/QA/Questions/1805/Many-to-many-realtionship-returning-null

### PR DESCRIPTION
**Pull request by EngincanV**
Resolves #9975

### CODE REVIEW

File: docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md [hunk 1]
```diff
@@ -0,0 +1,1572 @@
...
+
+![Demo](./demo.png)
+
+We need to see a page like in the above image. Our app is working properly, we can continue to development.
+
+> If you are stuck in any point, you can examine the [source codes](https://github.com/EngincanV/ABP-Many-to-Many-Relationship-Demo).
+
+#### Model Classes and Mapping Configurations
+
+Create a folder named **Modals** and add a class named `CategoryViewModel` inside of it. We will use this view modal class to determine which categories are selected or not in our Create/Edit modals.
```
Comment (enisn) [inline docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md]: I think it is `**Models**`

```suggestion
Create a folder named **Models** and add a class named `CategoryViewModel` inside of it. We will use this view modal class to determine which categories are selected or not in our Create/Edit modals.
```

File: docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md [hunk 2]
```diff
@@ -0,0 +1,1572 @@
...
+}
+```
+
+#### Create/Edit Modals
+
+After creating our index page for Books and configuring mappings, let's continue with creating the Create/Edit modals for Books.
+
+Create a razor page named **CreateModal.cshtml** (and **CreateModal.cshtml.cs**).
+
+* **CreateModal.cs**
```
Comment (enisn) [inline docs/en/Community-Articles/2021-10-31-Many-to-Many-Relationship-with-ABP-and-EF-Core/POST.md]:
```suggestion
* **CreateModal.cshtml**
```

**Review by enisn — CHANGES_REQUESTED** [review]
When I run the project at the first time. I couldn't see any category while creating a book. Also, the sample doesn't include a category creation page. So, what can I do? Should I create some categories into SQL manually or we should add something more into the example:

- A minimal data-seeder can be added to create some sample categories.
or
- A category creation page can be added into the example

**Comment by EngincanV** [comment]
@enisn I did not include a category page in the article on purpose. It would be longer if I included the category page. But it is a good idea to seed initial data for category and author, I will seed some initial data for them on my repo and also update the article.

**Comment by enisn** [comment]
Yeah, a minimal DataSeedContributor would be better to see how it works on the first run 🙂

**Review by enisn — APPROVED** [review]

**Review by enisn — APPROVED** [review]

**Pull request merged**